### PR TITLE
CC-7: List Page

### PR DESCRIPTION
**Pull request by Sarahunt89**
Created component projectCard.jsx
Created projectData.js
Screenshot CC-7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Jessica Dickerson - Team Lead
+Stephanie Cook - Team Member
+Sarah Ramos - Team Member
+
```
Comment (jdickerson-banyan) [inline README.md:4]: This file is not part of your ticket and should not be changed.

File: src/components/ProjectCard/style.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.cardDescription {
+    font-size: 20px;
+    font-weight: normal;
+}
+
+.cardImg {
+    display: flex;
+}
```
Comment (jdickerson-banyan) [inline src/components/ProjectCard/style.css:61]: Please add an extra line to this file

File: src/data/projectData.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        name: "Manifold",
+        description: "Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor accusantium sapienteveritatis accusamus explicabo eveniet impedit minima accusantium sapiente eius dignissimos harum!",
+        image: manifoldHomepage,
+    },
+    {
+        name: "ttv capital",
+        description: "Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor veritatis accusamus explicabo eveniet impedit minima accusantium sapiente eius dignissimos haru madi pisicing elit. Dolor veritatis accusamus.",
+        image: ttvHomepage,
+    },
+  ];
```
Comment (jdickerson-banyan) [inline src/data/projectData.js:21]: Please add all data points that are needed for each project

**Review by jdickerson-banyan — CHANGES_REQUESTED** [review]
Great job so far!

**Comment by Sarahunt89** [comment]
Added extra line to the bottom of style.css
Added all 8 data points to projectData.js

**Review by jdickerson-banyan — APPROVED** [review]

**Pull request merged**